ClassLoader/src/main/java/company/Helper.java

```diff
@@ -0,0 +1,53 @@
+package company;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Method;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.Scanner;
+
+public class Helper {
+
+    Class<?> aClass = null;
+    Logger log = LogManager.getLogger(Helper.class);
+    public Helper() {
+        while (true){
+            log("Menu: \n1-start \n0-exit \n");
+            Scanner scanner = new Scanner(System.in);
+            Integer chose = scanner.nextInt();
+            if(chose == 1) {
+                log("Input full path to jar: ");
+                String path = scanner.next();
+                log("Input jar name: ");
+                String jarName = scanner.next();
+                try {
+                    String url = "file://"+path+"/" + jarName;
+                    URL[] urlFromPath = new URL[]{new URL(url)};
+                    URLClassLoader loader = new URLClassLoader(urlFromPath);
```

Comment:
The Idea of this task was to use your own classloader not a standard one.
And the easiest way is to extend `URLClassLoader` by your own implementation (or just wrap it)